rootless-install.sh

```diff
@@ -307,6 +307,7 @@ print_instructions() {
 	echo "# Make sure the following environment variables are set (or add them to ~/.bashrc):\n"
 
 	if [ -n "$XDG_RUNTIME_DIR_CREATED" ]; then
+		echo "# WARN: systemd not found. You have to remove XDG_RUNTIME_DIR manually on every logout."
```

Comment:
Should we print the actual path (value of `XDG_RUNTIME_DIR`) instead (or "as well")?

Reply:
one line below :)